owsi-core/owsi-core-components/owsi-core-component-jpa-security/src/main/java/fr/openwide/core/jpa/security/password/rule/SecurityPasswordRules.java

```diff
@@ -0,0 +1,174 @@
+package fr.openwide.core.jpa.security.password.rule;
+
+import java.io.Serializable;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+
+import org.passay.CharacterRule;
+import org.passay.DictionaryRule;
+import org.passay.EnglishCharacterData;
+import org.passay.EnglishSequenceData;
+import org.passay.IllegalCharacterRule;
+import org.passay.IllegalRegexRule;
+import org.passay.IllegalSequenceRule;
+import org.passay.LengthRule;
+import org.passay.Rule;
+import org.passay.UsernameRule;
+import org.passay.WhitespaceRule;
+import org.passay.dictionary.ArrayWordList;
+import org.passay.dictionary.WordListDictionary;
+
+import com.google.common.collect.Lists;
+
+public class SecurityPasswordRules implements Serializable {
+
+	private static final long serialVersionUID = -2309617143631151956L;
+
+	private SecurityPasswordRules() {
+		super();
+	}
+
+	public static final Builder builder() {
+		return new Builder();
+	}
+
+	public static final class Builder {
+		
+		private Set<Rule> rules = new HashSet<Rule>();
```

Comment:
Agreed. It shouldn't require a major refactoring, only the build() method would be impacted.

Plus, the SecurityPasswordRules serves no purpose. You could just have a top-level `SecurityPasswordRulesBuilder` class and a `start()` static method.